__tests__/plugin-test.js

```diff
@@ -78,7 +78,12 @@ function buildPlugin(config = {}, _Plugin = TestPlugin) {
   // we work around the same relative issue by storing the commands executed,
   // and intercepting them to return replacement values (this is done in
   // execFormattedCommand just below)
-  container.shell.exec = Shell.prototype.exec;
+  container.shell.exec = (command, options, context) => {
```

Comment:
release-it v15 apparently no longer exports its `Shell` class nor the `format` helper, this solution is inspired by https://github.com/rwjblue/release-it-lerna-changelog/pull/208/files/e87f20015b70d1289d2a3c9c17e682109019954d#r905867552